tests/integration_tests/test_adc/test_adc.py

```diff
@@ -1,10 +1,9 @@
 """ Integration tests for EdgePi ADC module """
 
-
+# pylint: disable=wrong-import-order
```

Comment:
why not put the imports in the right order :)

Reply:
5f53f1f331b9cf4bad06e21dbe848486d20ec5b0